spec/fixtures/vcr_cassettes/webhook/create_invalid.yml

```diff
@@ -0,0 +1,75 @@
+---
+http_interactions:
+- request:
+    method: post
+    uri: http://staging-integration.posios.com/PosServer/rest/v1/webhooks/subscriptions
+    body:
+      encoding: UTF-8
+      string: '{"notificationUrl":""}'
+    headers:
+      User-Agent:
+      - excon/0.91.0
+      Accept:
+      - "*/*"
+      Content-Type:
+      - application/json
+      X-Auth-Token:
+      - API_TOKEN_VALUE
+  response:
+    status:
+      code: 400
+      message: Bad Request
+    headers:
+      Date:
+      - Mon, 02 Dec 2024 19:56:34 GMT
+      Content-Type:
+      - application/json;charset=UTF-8
+      Content-Length:
+      - '131'
+      Connection:
+      - keep-alive
+      X-Dns-Prefetch-Control:
```

Comment:
for a few of these cassettes, you can probably clean up some of this extra header info and have it still pass.